### DISCUSSION TITLE
Charging over night

### DISCUSSION THREAD

**Discussion by hexisXz** [InfiniTimeOrg/InfiniTime#1836] [General]
Is it okay to charge the watch at night? Will it effect the battery life, mess up the software reading the battery percentage or will the battery and the software be fine if I do so?

**Comment by SteveAmor** [comment]
I charge mine overnight and it's fine. Every 3 days prior to 1.13.0 and now every 9 or 10 days.

**Comment by JF002** [comment]
The charge is managed by a battery charge controller on board. It automatically stops and starts charging when needed, handles current/voltage protection,... so I guess that's fine to let the battery charge overnight (but I'm not a electronics engineer so maybe someone can confirm this?). I do it anyways ;)

On the software side, the battery level reported by InfiniTime won't be accurate while the watch is charging since the measured voltage will be higher during the charge, but other than that, it won't cause any issue.
